test/README.ssltest.md

```diff
@@ -89,6 +89,9 @@ handshake.
 
 * ExpectedTmpKeyType - the expected algorithm or curve of server temp key
 
+* ExpectedServerKeyType, ExpectedClientKeyType - the expected algorithm or
```

Comment:
Should be CertType not KeyType I think?

The client type isn't tested anywhere yet, is that right?